### DISCUSSION TITLE
Any example apps plotting df's etc?

### DISCUSSION THREAD

**Discussion by andrewm4894** [AnswerDotAI/fasthtml#605] [Q&A] [answered]
I'm wondering if there are any examples anywhere of an app that plots charts and data like time series data etc?

had a look here and did not see much: https://github.com/AnswerDotAI/fasthtml-example

context is im trying to port my simple streamlit app into a fancier fasthtrml app and make it turbo. 

this is the streamlit app using plotly: https://github.com/andrewm4894/anomstack/blob/main/dashboard.py

i have a feature branch here where i am messing around trying to get a sort of minimal fasthtml version together: https://github.com/andrewm4894/anomstack/blob/fasthtml/dashboard/main.py

i think going from plotly -> html should be easy enough. 

but was just wondering if there might be any examples out there already or maybe even some sort of better charting libraries that might take more advantage of the fullness of fasthtml.

eg am thinking of a little app that just generates some random time series and sort of plots each time series in a nice grid as cards and looks pretty. 

p.s. happy to try do this myself and contribute it to examples repo - just am new to the web stuff so not too confident at moment - still at the hill of the frustration/learning curve :)

**Comment by jamilraichouni** [comment, marked as answer]
Hi, this looks helpful: https://github.com/CarloLepelaars/fh-plotly

**Reply by andrewm4894** [reply]
Yep, it's actually used in the plotly example in here

https://gallery.fastht.ml/

Somehow I missed the gallery.
